packages/ove-service-auth/.dockerignore

```diff
@@ -0,0 +1,32 @@
+# visual studio code for node.js
+.vscode/
+
+# visual studio for .netcore
```

Comment:
Since this is a Python project you probably don't need this .NET stuff

Reply:
if you use vs code these files will be created anyway